src/ctia/http/routes/observable.clj

```diff
@@ -48,19 +46,28 @@
 
   (GET "/:observable_type/:observable_value/indicators" []
     :tags ["Indicator"]
-    :query [params IndicatorsByObservableQueryParams]
+    :query [params IndicatorRefsByObservableQueryParams]
     :path-params [observable_type :- ObservableType
                   observable_value :- s/Str]
-    :return (s/maybe [StoredIndicator])
-    :summary "Returns all the Indicators associated with the specified observable."
+    :return (s/maybe [c/Reference])
+    :summary "Returns all the Indicator References associated with the specified observable."
     :header-params [api_key :- (s/maybe s/Str)]
     :capabilities #{:list-judgements :list-indicators}
     (paginated-ok
-     (as-> {:type observable_type
-            :value observable_value} $
-       (list-judgements-by-observable @judgement-store $ nil)
-       (:data $ [])
-       (list-indicators-by-judgements @indicator-store $ params))))
+     (let [res (as-> {:type observable_type
+                      :value observable_value} $
+                 (list-judgements-by-observable @judgement-store $ nil)
+                 (:data $ [])
+                 (mapcat :indicators $)
+                 (map :indicator_id $)
+                 (set $) ;; remove duplicate URIs
+                 (vec $)
+                 (sort $))]
```

Comment:
Also I think you could just let the $ and move the rest of the sexprs into a the body.